local-certs/nats-server.conf

```diff
@@ -0,0 +1,5 @@
+authorization: {
+  users: [
+    { nkey: UAOG3WGHPVLI3OQQVVTAP33Q3VOJKHYRZWA7A3REHZ3AD4U7YJJUYXO4 }
```

Comment:
The default NATS NKey public key, required for LFH connect instances to communicate with each other. In the case where every LFH instance is owned by a separate entity, there will be one NKey for each LFH instance in this clause.